docs/docs/behaviors/layers.md

```diff
@@ -57,6 +57,14 @@ Example:
 &lt LOWER SPACE
 ```
 
+:::info
+Functionally, the layer-tap is a ["hold-preferred" flavor](hold-tap.md/#flavors) of [hold-tap](hold-tap.md) with a [`tapping-term-ms`](hold-tap.md/#tapping-term-ms) of 200 that takes in a [`momentary layer`](#momentary-layer) and a [keypress](key-press.md) as its "hold" and "tap" parameters respectively.
```

Comment:
```suggestion
Functionally, the layer-tap is a [hold-tap](hold-tap.md) of ["tap-preferred" flavor](hold-tap.md/#flavors) and a [`tapping-term-ms`](hold-tap.md/#tapping-term-ms) of 200 that takes in a [`momentary layer`](#momentary-layer) and a [keypress](key-press.md) as its "hold" and "tap" parameters, respectively.
```
This reads a bit better to me since I think of the flavor is a property of an hold-tap, rather than there existing N total hold-taps, one corresponding to each flavor. I am a bit nitpicking at this point though :)

Also fixed the flavor from hold- to tap-preferred: https://github.com/zmkfirmware/zmk/blob/main/app/dts/behaviors/layer_tap.dtsi